code/modules/psionics/faculties/coercion.dm

```diff
@@ -59,29 +59,45 @@
 	if(target.stat == DEAD || (target.status_flags & FAKEDEATH) || !target.client)
 		to_chat(user, SPAN_WARNING("\The [target] is in no state for a mind-read."))
 		return TRUE
+	if (iscarbon(target))
+		var/mob/living/carbon/C = target
+		var/obj/item/organ/internal/augment/psi/psiaug = C.internal_organs_by_name[BP_AUG_PSI]
+		if(psiaug && !psiaug.is_broken())
+			if(target.isSynthetic())
+				to_chat(user, SPAN_ALIEN("There is no mind here for you to dip your mentality into."))
+				return
+			else if (isvaurca(target))
```

Comment:
wouldn't a vaurca fitted with a psiaug be able to handle this?

Reply:
Yes, that does indeed work.